twrpdtgen/utils/find_package.py

```
@@ -0,0 +1,13 @@
+import os
+import platform
+
+def find_package(packageName):
+   result = []
+
+   for root, dir, package in os.walk("/"):
+      if packageName in package:
+         result = 1
+   return result
+
+if platform.system() == "Linux" and find_package("cpio") != 1:
+    print("cpio package is not installed\nInstall it by:\nsudo apt install cpio\nor\nsudo pacman -S cpio\n(Based on what package manager you're using)")
```

Logging feature exists, but here it's better to raise an exception

---

`raise RuntimeError("cpio not found")`

---

elaborated a bit.